### DISCUSSION TITLE
Replace MEETINGS.md with Google Doc?

### DISCUSSION THREAD

**Discussion by scottrigby** [gitops-working-group/gitops-working-group#123] [Proposals] [answered]
A real time collaborative document allows us to quickly organize meetings transparently, and make meeting notes available immediately as we work. It also allows participants to help pitch in on taking the notes. We are already using this Google doc for [GitOps Working Group Meetings](https://docs.google.com/document/d/1hxifmCdOV5_FbKloDJRWZQHq0ge-trXJKF-BgV4wHVk/edit?usp=sharing) because it solves these needs. We can have the markdown file in git too, but it takes extra manual work to sync it. Any objection to removing MEETINGS.md from this repo?

**Comment by chrisk700** [comment]
I found the MEETINGS.md a good jumping in point however the doc does appear to cover this and then some.

Would the "Working group meeting" link in README.md be updated to point to the doc?

**Reply by scottrigby** [reply]
You have a point. Definitely want to make participation and watching later as easy as possible for anyone interested...

**Comment by lloydchang** [comment, marked as answer]
@scottrigby Would it be acceptable to retain [MEETINGS.md](https://github.com/gitops-working-group/gitops-working-group/blob/main/MEETINGS.md) for its summary of Video Recording links, while [GitOps Working Group Meetings Google Doc](https://docs.google.com/document/d/1hxifmCdOV5_FbKloDJRWZQHq0ge-trXJKF-BgV4wHVk/) continues for real-time collaboration?  Until [GitOps Working Group Meetings Google Doc](https://docs.google.com/document/d/1hxifmCdOV5_FbKloDJRWZQHq0ge-trXJKF-BgV4wHVk/) includes all Video Recording links, [MEETINGS.md](https://github.com/gitops-working-group/gitops-working-group/blob/main/MEETINGS.md) shouldn't be replaced yet.  Thank you.

**Reply by scottrigby** [reply]
You and @chrisk700 make good points.

I just uploaded all the past meeting recordings we have to the YouTube channel CNCF made for the GitOps WG. See https://github.com/gitops-working-group/gitops-working-group/issues/71 issue description. Instead I'll update MEETINGS.md with this info.

Thanks!

**Reply by scottrigby** [reply]
@chrisk700 @lloydchang how does https://github.com/gitops-working-group/gitops-working-group/pull/133 look?

**Reply by lloydchang** [reply]
I added a suggestion at https://github.com/gitops-working-group/gitops-working-group/pull/133#discussion_r615209732
Thanks @scottrigby @chrisk700

**Reply by lloydchang** [reply]
Approved https://github.com/gitops-working-group/gitops-working-group/pull/133.

Thanks @chrisk700 @scottrigby
